### PR TITLE
fix: proper path when promise actor terminated quickly

### PR DESCRIPTION
**Pull request by Roiocam**
References #32330

### CODE REVIEW

**Comment by laglangyue** [comment]
LGTM

File: akka-actor-tests/src/test/scala/akka/pattern/AskSpec.scala [hunk 1]
```diff
@@ -256,6 +256,29 @@ class AskSpec extends AkkaSpec("""
...
+
+      val f = (act ? "ask").mapTo[String]
+      val promiseActorRef = p.expectMsgType[ActorRef]
+
+      // verify ask complete
+      val completed = f.futureValue
+      completed should ===("complete")
+      // verify path was proper
+      promiseActorRef.path.toString should include("pathPrefix")
+    }
```
Comment (johanandren) [inline akka-actor-tests/src/test/scala/akka/pattern/AskSpec.scala:281]: Fix looks good but I don't see how this tests asking a stopped actor?

Comment (johanandren) [reply akka-actor-tests/src/test/scala/akka/pattern/AskSpec.scala:281]: Ah, sorry, tricky test, we are returning sender to the outside, and the original ask already stopped. This is good.

**Review by johanandren — APPROVED** [review]

**Comment by johanandren** [comment]
Thanks!

**Pull request merged**